arcade_machine/games/arcade_settings.py

```diff
@@ -0,0 +1,70 @@
+
+from arcade_machine.games.game import Game
+from arcade_machine.sprites.label import Label
+from arcade_machine.components.music_player import music_volume, get_volume
+from arcade_machine.components.shape_to_sprite import rect_sprite
+from arcade_machine.events import CHANGE_GAME
+from pygame import font
+from pygame import draw
+from pygame import KEYDOWN, K_a, K_d, K_j, K_l, K_1, K_9, K_m
+from pygame import quit as pygame_quit
+from pygame.event import post as pygame_post_event
+from pygame.event import Event
+
+class ArcadeSettings(Game):
+    def __init__(self):
+        super().__init__()
+
+    def initialize(self):
+        self.background = (0, 0, 0)
+        self.volume = get_volume()
+
+        self.TITLE_FONT = font.Font('arcade_machine/resources/fonts/Main Menu/Early GameBoy.ttf', 48)
```

Comment:
This is fine for now but we'll likely want a better method of handling fonts.

Reply:
Fonts are custom to the specific game. This is easily done by setting some fonts in the init and using them throughout the update method.